app/__main__.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+"""Raw image data previewer - terminal functionality."""
+
+import argparse
+import os
+from .core import run_core_functionality
+
+parser = argparse.ArgumentParser(
+    prog=__package__,
+    description="preview raw data as an image of chosen format")
+parser.add_argument("FILE_PATH", help="file containing raw image data")
+parser.add_argument("-c",
+                    "--color_format",
+                    choices=["RGB3", "BGR3"],
```

Comment:
I would suggest connecting `choices` with some kind of variable (maybe map?) that will show every possible color_format.